Simplify creation of converter setups

tischi opened the pull request.

tischi committed:
Simplify creation of converter setups

tischi committed:
Introduce temporary variables, I don't know why, but it did not work without them...they should not be necessary

tischi commented:
@NicoKiaru 
For me the changes work (and in fact I need them for MoBIE, where we have quite unusual Converters...).

NicoKiaru commented:
Hi tischi,

Three small things and you can merge:

* the last commit is extremely weird : are you sure you can't find the reason ?
* since it's private methods, no need to keep them with deprecated annotation. The @deprecated annotation only makes sense for public methods.
* have you run the tests ? Do they all pass ?

tischi committed:
Remove unused private methods

tischi commented:
> the last commit is extremely weird : are you sure you can't find the reason ?

I changed it back and it also works (was something weird with IntelliJ...)


>    since it's private methods, no need to keep them with deprecated annotation. The @deprecated annotation only makes sense for public methods.

Good point! I did not realise they were private. I removed them.

>    have you run the tests ? Do they all pass ?

Yes.
